janus.go

```diff
@@ -555,6 +555,16 @@ type Session struct {
 	gateway *Gateway
 }
 
+//GetId ...
+func (session *Session) GetId() uint64 {
+	return session.ID
+}
+
+//ListenToEventsChan ...
+func (session *Session) ListenToEventsChan() chan interface{} {
```

Comment:
Could this just be Events()? The return type tells us it is a channel, and you don't *need* to listen to it, you can do whatever you want with it. Might be a good idea to return a `<-chan interface{}` though.